includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1,22 +1,56 @@
 <?php
-
-require_once( AMP__DIR__ . '/includes/sanitizers/class-amp-base-sanitizer.php' );
-
 /**
+ * Class AMP_Style_Sanitizer
+ *
  * Collects inline styles and outputs them in the amp-custom stylesheet.
  */
 class AMP_Style_Sanitizer extends AMP_Base_Sanitizer {
+
+	/**
+	 * @var string[] List of CSS styles in HTML content of DOMDocument ($this->dom)
+	 *
+	 * @since 0.4
+	 */
 	private $styles = array();
 
+	/**
+	 * Get list of CSS styles in HTML content of DOMDocument ($this->dom)
+	 *
+	 * @since 0.4
+	 *
+	 * @return string[]
+	 */
 	public function get_styles() {
+		if ( ! $this->did_convert_elements ) {
```

Comment:
Why this if `$this->styles` is already an `array()`?

Reply:
Consistency.  But to your point I guess it is not needed.